cluster/manifests/instana-poc/daemonset-instana.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: instana-agent
+    version: v1
+  name: instana-agent
+  namespace: kube-system
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: instana-agent
+      labels:
+        application: instana-agent
+    spec:
+      hostNetwork: true
+      hostIPC: true
+      hostPID: true
```

Comment:
nit: if you agree, I think these can now be grouped under the `securityContext` field which cleans it up a little (never tried that), see https://github.com/kubernetes/kubernetes/blob/e821e531db24ddd4897a166b67af8c863d93edc2/pkg/api/types.go#L2400

```yaml
spec:
  securityContext:
    hostNetwork: true
    hostIPC: true
    hostPID: true
```

Reply:
probably when we update to k8s >=1.8 we have to change this